File: projeto/unicafeWin/unicafe/src/main/java/br/edu/unilab/unicafe/model/Usuario.java
package br.edu.unilab.unicafe.model;

/**
 * 
 * @author Jefferson
 *
 */

public class Usuario {
	private int id;
	private String nome;
	private String email;
	private String login;
	private String senha;
	private int nivelAcesso;
	private String cpf;


	
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getLogin() {
		return login;
	}

	public void setLogin(String login) {
		this.login = login;
	}

	public String getSenha() {

		return senha;
	}

	public void setSenha(String senha) {
		this.senha = senha;
	}



	public Usuario(String nome, String email, String login, String senha, int nivelAcesso, String cpf) {

		this.nome = nome;
		this.email = email;
		this.login = login;
		this.senha = senha;
		this.setNivelAcesso(nivelAcesso);
		this.setCpf(cpf);
	}

	public Usuario(int id, String nome, String email, String login, String senha, int nivelAcesso, String cpf) {
		super();
		this.id = id;
		this.nome = nome;
		this.email = email;
		this.login = login;
		this.senha = senha;
		this.setNivelAcesso(nivelAcesso);
		this.setCpf(cpf);
	}

	public int getNivelAcesso() {
		return nivelAcesso;
	}

	public void setNivelAcesso(int nivelAcesso) {
		this.nivelAcesso = nivelAcesso;
	}

	public String getCpf() {
		return cpf;
	}

	public void setCpf(String cpf) {
		this.cpf = cpf;
	}

	public Usuario() {
		id = 0;
		nome = "";
		email = "";
		login = "";
		senha = "";
	}

	@Override
	public String toString() {
		// return "Usuario: id=" + id + "\n" + "nome=" + nome + "\n" + "email=" + email + "\n" + "login=" + login + "\n" + "senha=" + senha + "\n" + "nivelAcesso=" + nivelAcesso + "\n" + "cpf=" + cpf;
		return login;
	}

}

File: projeto/unicafeweb/unicafeWeb/teste/classes/sessao/Usuario.php
<?php


class Usuario{
	
	private $id;
	private $nome;
	private $email;
	private $login;
	private $senha;
	private $nivel;
	
	public function setId($id){
		$this->id = $id;
		
	}
	public function setNome($nome){
		$this->nome = $nome;
	}
	public function setEmail($email){
		$this->email = $email;
	}
	public function setLogin($login){
		$this->login =  $login;
	}
	public function setSenha($senha){
		$this->senha = $senha;
	}
	public function setNivel($nivel){
		$this->nivel = $nivel;
	}
	
	
	public function getId(){
		return $this->id;
	}
	public function getNome(){
		return $this->nome;
	}
	public function getEmail(){
		return $this->email;
	}
	public function getLogin(){
		return $this->login;
	}
	public function getSenha(){
		return $this->senha;
	}
	public function getNivel(){
		return $this->nivel;
	}
	
	
	
}

?>